lua/core/lualine/styles.lua

```diff
@@ -0,0 +1,221 @@
+    component_separators = "",
+    section_separators = "",
+    disabled_filetypes = {},
+  },
+  sections = {
+    lualine_a = {},
+    lualine_b = {},
+    lualine_c = {},
+    lualine_x = {},
+    lualine_y = {},
+    lualine_z = {},
+  },
+  inactive_sections = {
+    lualine_a = {},
+    lualine_b = {},
+    lualine_c = {},
+    lualine_x = {},
+    lualine_y = {},
+    lualine_z = {},
+  },
+  tabline = {},
+  extensions = {},
+}
+
+local default = {
+  style = "default",
+  options = {
+    icons_enabled = true,
+    component_separators = { "", "" },
+    section_separators = { "", "" },
+    disabled_filetypes = {},
+  },
+  sections = {
+    lualine_a = { "mode" },
+    lualine_b = { "branch" },
+    lualine_c = { "filename" },
+    lualine_x = { "encoding", "fileformat", "filetype" },
+    lualine_y = { "progress" },
+    lualine_z = { "location" },
+  },
+  inactive_sections = {
+    lualine_a = {},
+    lualine_b = {},
+    lualine_c = { "filename" },
+    lualine_x = { "location" },
+    lualine_y = {},
+    lualine_z = {},
+  },
+  tabline = {},
+  extensions = {},
+}
+
+local lvim = {
+  style = "lvim",
+  options = {
+    icons_enabled = true,
+    component_separators = "",
+    section_separators = "",
+    disabled_filetypes = { "dashboard", " ", "", "WhichKey" },
+  },
+  sections = {
+    lualine_a = {
+      {
+        function()
+          return " "
+        end,
+        left_padding = 0,
+        right_padding = 0,
+        condition = conditions.hide_in_width,
+      },
+    },
+    lualine_b = {
+      {
+        "branch",
+        icon = " ",
+        condition = conditions.check_git_workspace,
+      },
+    },
+    lualine_c = {
+      {
+        "diff",
+        symbols = { added = "  ", modified = "柳", removed = " " },
+        color_added = colors.green,
+        color_modified = colors.yellow,
+        color_removed = colors.red,
+        condition = conditions.hide_in_width,
+      },
+    },
+    lualine_x = {
+      {
+        "diagnostics",
+        sources = { "nvim_lsp" },
+        symbols = { error = " ", warn = " ", info = " ", hint = " " },
+        condition = conditions.hide_in_width,
+        color = { bg = colors.bg },
+      },
+      {
+        function()
+          if next(vim.treesitter.highlighter.active) ~= nil then
+            return "  "
+          end
+          return ""
+        end,
+        color = { fg = colors.green },
+        condition = conditions.hide_in_width,
+      },
+      {
+        function(msg)
+          msg = msg or "LSP Inactive"
+          local buf_clients = vim.lsp.buf_get_clients()
+          if next(buf_clients) == nil then
+            return msg
+          end
+          local buf_ft = vim.bo.filetype
+          local buf_client_names = {}
+          -- local null_ls_providers = require("lsp.null-ls").get_registered_providers_by_filetype(buf_ft)
+          local null_ls_providers = require("lsp.utils").get_active_client_by_ft(buf_ft)
```

Comment:
If what you want to show are the active language servers, then rename the variable accordingly :)